packages/source-map-utils/index.js

```diff
@@ -450,13 +450,21 @@ var SourceMapUtils = {
         return null;
       }
       index++;
-      while (instructions[index].name.match(/^PUSH\d*/)) {
+      while (
+        instructions[index] &&
+        instructions[index].name.match(/^PUSH\d*/)
```

Comment:
Should this be `/^PUSH\d+/` ?

Reply:
I mean, it doesn't really matter in this context?  There's no such instruction as plain `PUSH`.